_posts/courses/intro-to-earth-data-science-textbook/07-efficient-clean-code-python/loops/2019-10-22-loops-python-04-loops-exercise.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: single
```

Comment:
the name of this lesson i think should be 05 and not have "python" in it at the beginning. i think one lesson maybe didn't build or get deleted?

Reply:
thsi i think should be deleted